src/displayapp/screens/Notifications.cpp

```diff
@@ -166,6 +181,21 @@ Notifications::NotificationItem::NotificationItem(const char* title,
   lv_label_set_text_fmt(alert_count, "%i/%i", notifNr, notifNb);
   lv_obj_align(alert_count, NULL, LV_ALIGN_IN_TOP_RIGHT, 0, 16);
 
+  auto diff = std::chrono::system_clock::from_time_t(timeNow) - std::chrono::system_clock::from_time_t(timeArrived);
+  std::chrono::minutes age = std::chrono::duration_cast<std::chrono::minutes>(diff);
+  std::string ageString;
+  ageString.reserve(10);
+  if ((age.count() / (60 * 24)) >= 1) {
+    ageString = std::to_string(static_cast<uint16_t>(age.count() / (60 * 24))) + "d ago";
+  } else if ((age.count() / 60) >= 1) {
+    ageString = std::to_string(static_cast<uint8_t>(age.count() / 60)) + "h ago";
+  } else {
+    ageString = std::to_string(static_cast<uint8_t>(age.count())) + "m ago";
+  }
```

Comment:
`to_string` has been found to use a lot of memory #792. Using `sprintf` and char arrays might save some memory here too.

Reply:
Or maybe use lv_label_set_text_fmt for the int to string conversion?